src/unify.c

```diff
@@ -0,0 +1,45 @@
+#include <ucsan/unify.h>
+#include <ucsan/ucsan.h>
+
+// TODO: Should we use the cpp file?
+
+struct cached_info {
+	/* TODO: Add the data structure to handle mutiple items */
+	int dummy;
+};
+
+struct cached_info cached_info[NR_UCSAN_SLOT * NR_UCSAN_WP];
```

Comment:
If we want to discuss the container of how to maintain the task information, please create the issue. Not here.
Thanks.